vl-proza-message.src.js

```diff
@@ -1,6 +1,12 @@
-import {VlElement, define} from '/node_modules/vl-ui-core/vl-core.js';
+import { VlElement, define, awaitScript } from '/node_modules/vl-ui-core/vl-core.js';
 import '/node_modules/vl-ui-button/vl-button.js';
 import '/node_modules/vl-ui-icon/vl-icon.js';
+import '/node_modules/vl-ui-typography/vl-typography.js';
+
+awaitScript('tinymce', '/node_modules/tinymce/tinymce.min.js').then(() => {
```

Comment:
Kan dit niet beter met module imports? https://www.tiny.cloud/docs/advanced/usage-with-module-loaders/

Reply:
Nee, we willen de define pas aanroepen nadat de TinyMCE lib geladen is, dat is niet mogelijk met module imports?

Reply:
Is het niet per definitie zo dat een module import gaat wachten om de rest van de code uit te voeren? Want anders werkt heel dat systeem toch niet: je importeert iets van een library en hij gaat verder zonder wachten. Dan kan hij toch een referentie naar iets van de library tegenkomen die nog niet ingeladen is? 

Zie ook https://exploringjs.com/impatient-js/ch_modules.html#example-loading-a-module-dynamically bijvoorbeeld, om het op een dynamische manier te doen met module imports.

Reply:
Als er een export zou zijn inderdaad, maar lijkt blijkbaar niet te werken: "Uncaught SyntaxError: The requested module '/node_modules/tinymce/tinymce.js' does not provide an export named 'default'".

Reply:
Blijkbaar werkt een niet named import wel, dus ik pas het aan.